lighthouse-core/gather/driver/network-monitor.js

```diff
@@ -106,10 +124,17 @@ class NetworkMonitor {
     this._networkRecorder.on('requestloaded', reEmit('requestloaded'));
 
     this._session.on('Page.frameNavigated', this._onFrameNavigated);
-    this._targetManager.addTargetAttachedListener(this._onTargetAttached);
-
     await this._session.sendCommand('Page.enable');
-    await this._targetManager.enable();
+
+    // Legacy driver does its own target management.
+    // @ts-expect-error
+    const isLegacyRunner = Boolean(this._session._domainEnabledCounts);
+    if (!isLegacyRunner) {
```

Comment:
super nit but i'd switch the order of these blocks... so instead: `if (isLegacyRunner) ... else ...`

just feels more natural since your `is` case isnt instantly negated.